getMongoData/getMongoData.js

```diff
@@ -310,11 +310,12 @@ function printDataInfo(isMongoS) {
                                     {$indexStats: {}},
                                     {$group: {_id: "$key", stats: {$push: {accesses: "$accesses.ops", host: "$host", since: "$accesses.since"}}}},
                                     {$project: {key: "$_id", stats: 1, _id: 0}}
-                                  ]
+                                  ],
+                                  cursor: {}
                                 });
 
-                                if (res.hasOwnProperty('result')) {
-                                  res.result.forEach(
+                                if (res.hasOwnProperty('cursor') && res.cursor.hasOwnProperty('firstBatch')) {
```

Comment:
Please add a comment which explains that there will only ever be a single batch, due to (a) the way the aggregation groups on the index key and (b) the max 64 indexes per collection limit.